dataland-frontend/tests/component/components/pages/MyDataRequestsOverview.cy.ts

```diff
@@ -0,0 +1,212 @@
+import { type ExtendedStoredDataRequest, RequestStatus } from "@clients/communitymanager";
+import RequestedDatasetsPage from "@/components/pages/MyDataRequestsOverview.vue";
+import { minimalKeycloakMock } from "@ct/testUtils/Keycloak";
+import { DataTypeEnum } from "@clients/backend";
+
+const mockDataRequests: ExtendedStoredDataRequest[] = [];
+const expectedHeaders = ["COMPANY", "REPORTING PERIOD", "FRAMEWORK", "REQUESTED", "LAST UPDATED", "STATUS"];
+
+before(function () {
+  mockDataRequests.push({
+    dataRequestId: "dummyId",
+    datalandCompanyId: "compA",
+    companyName: "companyAnswered",
+    dataType: DataTypeEnum.P2p,
+    reportingPeriod: "2020",
+    creationTimestamp: 1709204495770,
+    lastModifiedDate: 1709204495770,
+    requestStatus: RequestStatus.Answered,
+  } as ExtendedStoredDataRequest);
+  mockDataRequests.push({
+    dataRequestId: "dummyId",
+    datalandCompanyId: "compO",
+    companyName: "companyNotAnswered1",
+    dataType: DataTypeEnum.Sme,
+    reportingPeriod: "2022",
+    creationTimestamp: 1709204495770,
+    lastModifiedDate: 1709204495770,
+    requestStatus: RequestStatus.Open,
+  } as ExtendedStoredDataRequest);
+  mockDataRequests.push({
+    dataRequestId: "dummyId",
+    datalandCompanyId: "compC",
+    companyName: "zcompany",
+    dataType: DataTypeEnum.EutaxonomyFinancials,
+    reportingPeriod: "3021",
+    creationTimestamp: 1809204495770,
+    lastModifiedDate: 1609204495770,
+    requestStatus: RequestStatus.Closed,
+  } as ExtendedStoredDataRequest);
+  mockDataRequests.push({
+    dataRequestId: "dummyId",
+    datalandCompanyId: "compC",
+    companyName: "companyNotAnswered3",
+    dataType: DataTypeEnum.EutaxonomyNonFinancials,
+    reportingPeriod: "2021",
+    creationTimestamp: 1709204495770,
+    lastModifiedDate: 1709204495770,
+    requestStatus: RequestStatus.Open,
+  } as ExtendedStoredDataRequest);
+  mockDataRequests.push({
+    dataRequestId: "dummyId",
+    datalandCompanyId: "compC",
+    companyName: "acompany",
```

Comment:
Rename to sth more understandable: maybe 
`A-company-that-will-always-be-sorted-to-top`
`Z-company-that-will-always-be-sorted-to-bottom`